### PR TITLE
[Doc] Minor fix to MKP secret setup guide

### PR DESCRIPTION
**Pull request by numerology**
Change the way of specifying namespace.

Sometimes `--current` option might not work for gcloud in MacOS, causing empty secret and permission issues.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2576)

### CODE REVIEW

File: manifests/gcp_marketplace/guide.md [hunk 1]
```diff
@@ -59,11 +59,8 @@ and store the service account credential as a Kubernetes secret `user-gcp-sa` in
 # Create credential for the service account
 gcloud iam service-accounts keys create application_default_credentials.json --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com
 
-# Make sure the secret is created under the correct namespace.
-kubectl config set-context --current --namespace=$NAMESPACE
-
 # Attempt to create a k8s secret. If already exists, override.
-kubectl create secret generic user-gcp-sa \
+kubectl create secret generic user-gcp-sa -n $NAMESPACE\
```
Comment (Bobgy) [inline manifests/gcp_marketplace/guide.md]: Minor usability concern, copying a command of 3 lines and then modifying something on the first line is hard.
I prefer
```
kubectl create secret generic user-gcp-sa \
--from-file=user-gcp-sa.json=application_default_credentials.json \
-n $NAMESPACE --dry-run -o yaml  |  kubectl apply -f -
```

**Comment by rmgogogo** [comment]
Thanks, give hours to do more test on it.
I find something wrong in my existing cluster, wondering whether we have other issue. (not sure whether you tested the new CLI)

File: manifests/gcp_marketplace/guide.md (hunk 1 above)
Comment (numerology) [reply manifests/gcp_marketplace/guide.md]: Done.

**Comment by numerology** [comment]
> Thanks, give hours to do more test on it.
> I find something wrong in my existing cluster, wondering whether we have other issue. (not sure whether you tested the new CLI)

What's the symptom? I did not try out the new cli.

File: manifests/gcp_marketplace/guide.md [hunk 2]
```diff
@@ -59,13 +59,10 @@ and store the service account credential as a Kubernetes secret `user-gcp-sa` in
 # Create credential for the service account
 gcloud iam service-accounts keys create application_default_credentials.json --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com
```
Comment (rmgogogo) [inline manifests/gcp_marketplace/guide.md]: gcloud iam service-accounts keys create application_default_credentials.json \
  --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com

how about also split it into 2 lines for easy copy?

**Comment by rmgogogo** [comment]
tested and all works well.

/lgtm

**Comment by Bobgy** [comment]
@IronPan for approval

File: manifests/gcp_marketplace/guide.md (hunk 2 above)
Comment (numerology) [reply manifests/gcp_marketplace/guide.md]: Done

**Comment by numerology** [comment]
k8s robot seems stop working, manually merged

**Pull request merged**